charts/ingress-nginx/values.yaml

```
@@ -85,6 +85,8 @@ controller:
   enableTopologyAwareRouting: false
   # -- This configuration disable Nginx Controller Leader Election
   disableLeaderElection: false
+  # -- This configuration set leaderElection Time to Live
```

Well, of course it does set that. But what's the effect of changing that value? 😉

```suggestion
  # -- Duration a leader election is valid before it's getting re-elected
```

Also, please put this value right below `electionId` if renaming it.